css-canvas-medical/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Challenge - Canvas Medical</title>
+    <link rel="stylesheet" href="fonts.css" />
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <section id="nav-section">
+      <div class="row nav-bar">
+        <div class="column-one-fifth logo">
+          <img src="images/logo-blue.svg" alt="canvas logo" id="logo-blue" />
+        </div>
+        <div class="column-fill"></div>
+        <div class="column-one-half nav-links">
+          <a href="" class="nav-button">Product</a>
+          <a href="" class="nav-button">Implementation</a>
+          <a href="" class="nav-button">About Us</a>
+          <a href="" class="nav-button">Blog</a>
+          <a href="" class="nav-button" id="contact-button">Contact Us</a>
+        </div>
+      </div>
+    </section>
+    <div class="border"></div>
+    <section>
+      <div class="row">
```

Comment:
Instead of having a `div` for your border you could use `border-bottom` property on your `.nav-bar` section or `#nav-section` and used some padding to space it lower